loleaflet/css/toolbar.css

```diff
@@ -1052,3 +1052,84 @@ menu-entry-with-icon.padding-left + menu-entry-icon.width */
 .menu-entry-no-icon {
 	padding-left: 42px;
 }
+
+.userInfo {
+	border-radius: 100%;
+	display: flex;
+	justify-content: space-between;
+	align-items: baseline;
+}
+
+.userInfo-content {
+	display: none;
+	position: absolute;
+	padding: 12px 16px;
+	z-index: 1;
+	margin-top: 50px;
+	left: -130px;
+	color: #222;
+	background-color: #fff;
+	border-radius: 3px;
+	filter: drop-shadow(0 1px 3px rgba(77, 77, 77, 0.5));
+	min-width: 150px;
+}
+
+.userInfo:hover .userInfo-content {
+	display: block;
+}
+
+.document-title {
+	justify-content: space-between;
+}
+
+#user-avt {
+	width: 35px;
+	height: 35px;
+	border-radius: 100%;
+}
+
+.userInfo-content::after {
+	border: 10px solid transparent;
+	border-bottom-color: var(--color-main-background);
+	bottom: 100%;
+	content: ' ';
+	height: 0;
+	width: 0;
+	position: absolute;
+	pointer-events: none;
+	color: #f9f9f9;
+	right: 25px;
+}
+
+.user-item-wrapper {
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.user-item {
+	background-position: center;
+	background-size: cover;
+	background-repeat: no-repeat;
+	border-width: 2px;
+	border-style: solid;
+	width: 25px;
+	height: 25px;
+	border-radius: 100%;
+	margin-right: 5px;
+}
+
+.user-top {
+	width: 25px;
+	height: 25px;
+	background-color: white;
+	border-radius: 100%;
+	margin-right: -10px;
+	background-size: cover;
+	background-repeat: no-repeat;
+	border-style: solid;
+}
+
+.guess {
+	opacity: 0.5;
```

Comment:
I'd say we don't need a separation between the display of the current user or guest like that. Also opacity may look a bit odd when other avatars are shining through